src/platforms/react-native/manual-setup/ram-bundles.mdx

```diff
@@ -12,6 +12,12 @@ All the existing RAM bundle formats are explained in detail in the [Metro Bundle
 
 ## Enabling RAM Bundles
 
+<Note>
+
+If you are using [Hermes](/platforms/react-native/manual-setup/hermes/), you should not have RAM bundles feature enabled. In Hermes, when loading the bytecode, `mmap` ensures that the entire file is not loaded. Using Hermes with RAM bundles might lead to issues, because those mechanisms are not compatible with each other.
```

Comment:
```suggestion
If you are using [Hermes](/platforms/react-native/manual-setup/hermes/), you should not have the RAM bundles feature enabled. In Hermes, when loading the bytecode, `mmap` ensures that the entire file is not loaded. Using Hermes with RAM bundles might lead to issues, because those mechanisms are not compatible with each other.
```